### PR TITLE
Vs code api helper changes

### PR DESCRIPTION
**Pull request by abigailsmith97**
### JIRA link (if applicable) ###
N/A

### Change description ###
Changes to the api helper and config due to clashes in how intelliJ and vscode read the files.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by GauravChawlaMOJ — APPROVED** [review]

File: playwright-e2e/settings/user-credentials.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import dotenv from "dotenv";
 dotenv.config();
 
-const e2ePw = process.env.E2E_TEST_PASSWORD || '';
-const defaultPwd = process.env.SYSTEM_UPDATE_USER_PASSWORD || '';
+const e2ePw = process.env.E2E_TEST_PASSWORD || 'Password1234';
```
Comment (Iphiclus) [inline playwright-e2e/settings/user-credentials.ts:4]: Hi, do we need the passwords hardcoded?

Comment (GauravChawlaMOJ) [reply playwright-e2e/settings/user-credentials.ts:4]: May be not, if it fails to read environment variable in IDE, then add it into your local copy of project only and not commit? Also, fortify scan would catch this as a critical issue I suppose.

File: playwright-e2e/utils/api-helper.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export class Apihelp {
           'Content-Type': 'application/x-www-form-urlencoded',
         },
       };
-      let url = `${urlConfig.idamUrl}loginUser?username=${user.email}&password=${user.password}`;
+      let url = `https://idam-api.aat.platform.hmcts.net/loginUser?username=${user.email}&password=${user.password}`;
```
Comment (DanCatchpole) [inline playwright-e2e/utils/api-helper.ts:18]: is there any reason this was changed from the config value, as would break tests against demo?

Comment (Iphiclus) [reply playwright-e2e/utils/api-helper.ts]: fixed

**Review by DanCatchpole — APPROVED** [review]
lgtm!

**Review by Iphiclus — APPROVED** [review]

**Pull request merged**